sympy/physics/mechanics/body.py

```diff
@@ -155,6 +161,20 @@ def z(self):
         """The basis Vector for the Body, in the z direction. """
         return self.frame.z
 
+    def kinetic_energy(self, frame):
+        """Kinetic energy of the body."""
```

Comment:
Make full doc string, also should take ReferenceFrame or Body.

Reply:
Added